doc.php

```diff
@@ -35,26 +35,28 @@
   }
   if (!isset($repository))
     $repository = 'omichel';
+  $url = "https://raw.githubusercontent.com/$repository/webots-doc/";
+  $rawgiturl = "https://cdn.rawgit.com/$repository/webots-doc/";
```

Comment:
We could use the CDN only for the master branch, in order to increase the update time.